hack/e2e-internal/e2e-up.sh

```diff
@@ -29,16 +29,40 @@ source "${KUBE_ROOT}/cluster/kube-util.sh"
 
 prepare-e2e
 
+function calc_next_cidr {
```

Comment:
Alright, conceded. Here is the dumbed down version - https://github.com/kubernetes/kubernetes/pull/27892